aiohttp/_websocket/reader_py.py

```diff
@@ -458,8 +458,7 @@ def _feed_data(self, data: bytes) -> None:
                     self._payload_fragments.append(data_cstr[f_start_pos:f_end_pos])
                     if self._has_mask:
                         assert self._frame_mask is not None
-                        payload_bytearray = bytearray()
-                        payload_bytearray.join(self._payload_fragments)
+                        payload_bytearray = bytearray().join(self._payload_fragments)
```

Comment:
Sadly this was an simple refactoring error which I didn't notice because the coverage here wasn't actually testing this case. We now have coverage for this.